### DISCUSSION TITLE
Device Pixel Ratio

### DISCUSSION THREAD

**Discussion by AFPass** [Zefau/ioBroker.jarvis#1269] [Q&A]
Hallo,

was bedeutet "Device Pixel Ratio" in den Einstellungen?
<img width="333" alt="image" src="https://user-images.githubusercontent.com/13101074/145037549-c9b6b3aa-13a4-48aa-b6b6-db1363c67ce7.png">

**Comment by Zefau** [comment]
Bitte für Fragen immer die Discussions nutzen.

**Comment by Zefau** [comment]
Werden dir untersvhiedliche Werte bei Tablet, Handy und Desktop gezeigt?

**Comment by mcuiobroker** [comment]
Bei mir auf dem PC:
![image](https://user-images.githubusercontent.com/84818934/145037978-f8cee3e8-09d1-4261-8511-cb6b77eca9d8.png)
Beim Tablet und Handy kann man nicht in die Konfig schauen!
Vielleicht wäre eine Option in den Einstellungen sinnvoll, um es machen zu können?

**Reply by AFPass** [reply]
Einfach https:// < ioBrokerServer > :8082/jarvis/index.html#/0/configuration/settings aufrufen

**Comment by AFPass** [comment]
Auf dem Mac: 2
Auf dem iPhone: 3

**Comment by Zefau** [comment]
Hat noch jemand ein Tablet zum Vergleich?

**Reply by mcuiobroker** [reply]
Tablet Samsung Galaxy Tab A Device-Pixel: 1.5

**Reply by mcuiobroker** [reply]
Tablet Samsung TAB A6 Device-Pixel: 1.5

**Comment by AFPass** [comment]
iPad 6: 2

**Reply by Bernd9000** [reply]
Xiaomi MI9T  2.72727272727272
Lenovo Smart Tab 1.5
